MaskRCNN/eval.py

```diff
@@ -59,7 +60,7 @@ def _scale_box(box, scale):
     scaled_box[3] = y_c + h_half
     return scaled_box
 
-
+@jit
```

Comment:
Not necessary for this PR, but we should add in explicit types for the jit, that should give us a slight performance boost and get rid of most of the warnings.